tests/model_registry/model_catalog/plugin_arch/test_unified_catalog_server.py

```
@@ -0,0 +1,68 @@
+        catalog_base_url: str,
+        model_registry_rest_headers: dict[str, str],
+        endpoint: str,
+        expected_status: str,
+        expect_plugins: bool,
+    ):
+        """Given the catalog server is running with registered plugins
+        When querying a health endpoint
+        Then the server reports the expected status
+        """
+        response = requests.get(
+            f"{catalog_base_url}/{endpoint}", headers=model_registry_rest_headers, verify=False, timeout=30
```

Ruff (0.15.14)

[error] Probable use of `requests` call with `verify=False` disabling SSL certificate checks

(S501)